server.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express")
+const app = express()
+const port = 3000
+
+app.get('/ping',(req,res)=>{
```

Comment:
It's a good practice to handle errors and send appropriate status codes in the response. Consider adding error handling in your routes. [medium]